pkg/server/commands/list_objects.go

```diff
@@ -57,9 +57,24 @@ type ListObjectsQuery struct {
 	checkResolver graph.CheckResolver
 }
 
+type ResolutionMetadata struct {
```

Comment:
```suggestion
type ListObjectsResolutionMetadata struct {
```

Reply:
Because `ListObjectsQuery` is not scoped inside of it's own pkg. So let's be clear what the metadata applies to.